src/scripts/site.js

```diff
@@ -1,216 +1,333 @@
-            require(["connectionManagerFactory", "apphost", "credentialprovider", "events", "userSettings"], function(ConnectionManager, apphost, credentialProvider, events, userSettings) {
-                var credentialProviderInstance = new credentialProvider;
-                var promises = [apphost.getSyncProfile(), apphost.init()];
-                Promise.all(promises).then(function(responses) {
-                    var deviceProfile = responses[0],
-                        capabilities = Dashboard.capabilities(apphost);
-                    capabilities.DeviceProfile = deviceProfile;
-                    var connectionManager = new ConnectionManager(credentialProviderInstance, apphost.appName(), apphost.appVersion(), apphost.deviceName(), apphost.deviceId(), capabilities, window.devicePixelRatio);
-                    if (defineConnectionManager(connectionManager), bindConnectionManagerEvents(connectionManager, events, userSettings), !AppInfo.isNativeApp) return console.log("loading ApiClient singleton"), getRequirePromise(["apiclient"]).then(function(apiClientFactory) {
+        return require(["connectionManagerFactory", "apphost", "credentialprovider", "events", "userSettings"], function (ConnectionManager, apphost, credentialProvider, events, userSettings) {
+            var credentialProviderInstance = new credentialProvider(),
+                promises                   = [apphost.getSyncProfile(), apphost.init()];
+
+            Promise.all(promises).then(function (responses) {
+                var deviceProfile = responses[0],
+                    capabilities  = Dashboard.capabilities(apphost);
+
+                capabilities.DeviceProfile = deviceProfile;
+
+                var connectionManager = new ConnectionManager(credentialProviderInstance, apphost.appName(), apphost.appVersion(), apphost.deviceName(), apphost.deviceId(), capabilities, window.devicePixelRatio);
+
+                defineConnectionManager(connectionManager);
+                bindConnectionManagerEvents(connectionManager, events, userSettings);
+
+                if (!AppInfo.isNativeApp) {
+                    console.log("loading ApiClient singleton");
+
+                    return require(["apiclient"], function (apiClientFactory) {
                         console.log("creating ApiClient singleton");
+
                         var apiClient = new apiClientFactory(Dashboard.serverAddress(), apphost.appName(), apphost.appVersion(), apphost.deviceName(), apphost.deviceId(), window.devicePixelRatio);
-                        apiClient.enableAutomaticNetworking = !1, apiClient.manualAddressOnly = !0, connectionManager.addApiClient(apiClient), window.ApiClient = apiClient, localApiClient = apiClient, console.log("loaded ApiClient singleton"), resolve()
+                        
+                        apiClient.enableAutomaticNetworking = false;
+                        apiClient.manualAddressOnly         = true;
```

Comment:
I kind of get what you're going for here, but I'd prefer not having all this extra whitespace. If you add a variable with a longer name, you'll need to indent everything etc.
```suggestion
                        apiClient.manualAddressOnly = true;
```